pkg/services/sqlstore/annotations_cleanup.go

```diff
@@ -0,0 +1,49 @@
+package sqlstore
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/grafana/grafana/pkg/bus"
+	m "github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/util/errutil"
+)
+
+func init() {
+	bus.AddHandler("sql", deleteExpiredAnnotations)
+}
+
+const MAX_EXPIRED_ANNOTATIONS_TO_DELETE = 900
```

Comment:
Why 900?

Reply:
I'm also unsure about that @marefr. Maybe we should derive a sensible number?

Reply:
Initially tried 1000 but due to unit tests failures changed to 900